main.rb

```diff
@@ -0,0 +1,64 @@
+require 'json'
+require_relative 'App/Game/game'
+require_relative 'App/Author/author'
+require_relative 'App/Item/item'
+require_relative 'App/MusicAlbum/music_album'
+require_relative 'App/Book/book_label'
+
+MusicAlbum.load_albums_from_json
+
+book_label = BookLabel.new
+# ALBUMS_FILE = 'albums.json'.freeze
+# MOVIES_FILE = 'movies.json'.freeze
+# GAMES_FILE = 'games.json'.freeze
+# AUTHORS_FILE = 'authors.json'.freeze

```

Comment:
- [ ] @Team it's not good practice to have commented code in your codebase, I would recommend you remove the commented code.

Reply:
finished! ✔️